nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -66,27 +69,50 @@ where
         Self: Sized,
     {
         Ok(Self {
-            settings: Arc::new(settings),
-            _cl: core::marker::PhantomData,
+            settings,
+            _tx: core::marker::PhantomData,
+            _storage_serde: core::marker::PhantomData,
         })
     }
 
     async fn serve(self) -> Result<(), Self::Error> {
-        let store = self.settings.clone();
         let app = Router::new()
             .merge(SwaggerUi::new("/swagger-ui").url("/api-docs/openapi.json", ApiDoc::openapi()))
             .route("/da/metrics", routing::get(da_metrics))
             .route("/da/status", routing::post(da_status))
-            .route("/cl/metrics", routing::get(cl_metrics::<ClTransaction>))
-            .route("/cl/status", routing::post(cl_status::<ClTransaction>))
+            .route("/da/blobs", routing::post(da_blobs))
+            .route("/cl/metrics", routing::get(cl_metrics::<T>))
+            .route("/cl/status", routing::post(cl_status::<T>))
+            .route("/carnot/info", routing::get(carnot_info::<T, S, SIZE>))
+            .route("/network/info", routing::get(libp2p_info))
+            .route("/storage/block", routing::post(block::<S, T>))
+            .route("/mempool/add/tx", routing::post(add_tx::<T>))
+            .route("/mempool/add/cert", routing::post(add_cert))
+            .with_state(self.settings.handle);
```

Comment:
Why do we have all this here now? this was merged right?

Reply:
I merged other approved PR to this PR, this PR has not been merged.